change/@microsoft-teams-js-f6daee5c-d395-44ed-a4f3-670c3bbc44fe.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added DataLayer capability that will allow Apps to leverage data layer in the Host. The capability is still awaiting support in one or most host applications. To track availability of this capability across different hosts see https://aka.ms/capmatrix",
```

Comment:
Should we change this to be clear that this isn't a generally available capability? I am sure this feature will not be able to be tracked in the public https://aka.ms/capmatrix...

On my telemetry PR my change file reads "Added private MessageChannels capability".